config/locales/hr/organisations.yml

```diff
@@ -114,6 +114,9 @@ hr:
           many:
           one: Nacrt teksta
           other: Nacrt tekstova
+        drcf_digital_markets_research:
```

Comment:
This needs `few` and `many` as well

Reply:
This isn't related to the new templates, so addressed in a separate PR: https://github.com/alphagov/collections/pull/2779